tox.ini

```diff
@@ -3,6 +3,7 @@ envlist =
     py{38,37,36,35,27}
     docs
     style
+    nightly
```

Comment:
you don't need to add this to `envlist`, this is just the list of things that get iterated through when you enter `tox` (with no `-e`). It will still be available to run without it.

Reply:
ah, good to know!